src/unix/bsd/freebsdlike/freebsd/mod.rs

```diff
@@ -1249,6 +1249,8 @@ f! {
 }
 
 extern {
+    pub fn gettimeofday(tp: *mut ::timeval,
```

Comment:
Why not combine FreeBSD's and Dragonfly's definitions in src/unix/bsd/freebsdlike/mod.rs ?